README.md

```diff
@@ -24,18 +24,18 @@ In order to run this Web App locally you will need to install:
 
 Key                | Description
 -------------------|------------------------------------
-`LFP_PAY_WEB_PORT` |The port of the LFP-PAY-WEB service
+`PENALTY_PAYMENT_WEB_PORT` |The port of the Penalty-Payment-WEB service
```

Comment:
If we're changing this Env variable in the `start.sh` top level file - I think that is intended for other environments.  Will you be raising a PR to change this new Env variable in our local docker please?
* https://github.com/companieshouse/docker-chs-development/blob/feature/austin/sanctions/services/modules/lfp-appeals/pps-pay-web.docker-compose.yaml#L31

Reply:
Branch created - https://github.com/companieshouse/docker-chs-development/tree/feature/austin/SAN-48-PPS-Web-Rename

Reply:
The PR is created 
https://github.com/companieshouse/docker-chs-development/pull/1497
And the branch is 
https://github.com/companieshouse/docker-chs-development/tree/feature/austin/SAN-48-PPS-Web-Rename-Env

Reply:
Approved - happy for you to merge https://github.com/companieshouse/docker-chs-development/pull/1497 once the repos are renamed and this PR gets merged.  Thanks